css/src/components/checkbox.scss

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1em !default;
```

Comment:
I've just put a comment in the Figma related to this, but I think we should keep the text size of the default tied to 1rem, which to keep relatively similar proportions means we'd want to update the following to variables:

```
$checkbox-size: 1.25em !default;
$checkbox-icon-size: 0.875em !default;

Reply:
Reverted back to the sizes in your comment.